VideoApi/Testing.Common/Helper/Builders/Domain/ConferenceBuilder.cs

```diff
@@ -177,8 +177,8 @@ public ConferenceBuilder WithInterpreterRoom()
             {
                 WithParticipants(2);
             }
-            var room = new Builder(_builderSettings).CreateNew<InterpreterRoom>().WithFactory(() =>
-                new InterpreterRoom(_conference.Id, "InterpreterRoom1", VirtualCourtRoomType.Civilian)).Build();
+            var room = new Builder(_builderSettings).CreateNew<ParticipantRoom>().WithFactory(() =>
+                new ParticipantRoom(_conference.Id, "InterpreterRoom1", VirtualCourtRoomType.Civilian)).Build();
```

Comment:
Would something like LinkedRoom or LinkedParticipantRoom be clearer as to the intent? A consultation room also has participants but does not fall under this right?

Reply:
The reason it's called a participant room because the room is treated as a participant in a hearing. Linked wouldn't be appropriate because Judicial Office Holders are not linked

Reply:
I started the session yesterday by explaining the reason behind the refactor and why it was renamed to ParticipantRom

Reply:
> The reason it's called a participant room because the room is treated as a participant in a hearing. Linked wouldn't be appropriate because Judicial Office Holders are not linked

Okay. Makes sense.